examples/with-nx/README.md

```diff
@@ -0,0 +1,90 @@
+# Example app using Nx
+
+**NOTE:** When using Nx on Vercel, ensure you are using `nx@14.6.2` or above.
+
+This example was created using [Nx](https://nx.dev).
+
```

Comment:
Do we have a deployed example we can add here?

Reply:
We don't right now but I can add one.